docker-compose.yml

```diff
@@ -7,9 +7,7 @@ services:
       - MYSQL_USER=remo
       - MYSQL_PASSWORD=remo
       - MYSQL_ROOT_PASSWORD=root
-    entrypoint:
-      - docker-entrypoint.sh
-      - --max_allowed_packet=2M
+    command: mysqld --max_allowed_packet=64M
```

Comment:
Not exist in this one, I am not sure if generated by docker but in any case in my installation I got an error about it and with fix now is working.